### DISCUSSION TITLE
guide/webdav

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#43] [Announcements]
# guide/webdav

Documentation for alist v3

https://alist.nn.ci/guide/webdav.html

**Comment by Dashan-IZ** [comment]
谁能告诉我，v3版本的webdav怎么配置，我已经找了一个星期资料，就是没有办法链接

**Reply by alrightw** [reply]
不知道你是用什么app加载，如果用kodi的话注意有webdav（http）、webdav（https）两种，注意区分。kodi、nplay、infuse加载路径要填dav 端口默认5244

**Reply by RichardQian** [reply]
我遇到的情况是admin不成功，建个别的用户就成功了

**Reply by manyouzhe** [reply]
我也一直链接不成功，不知道咋回事

**Comment by CrazyMouse** [comment]
很奇怪，ios nplayer 可以看列表，但是一点视频播放就是403，并且也无法下载

**Reply by photor** [reply]
我的问题类似，用rclone挂载后文件列表没有问题，但一复制就是403。你解决了吗？

**Reply by CrazyMouse** [reply]
搞定了 webdav策略设置成“本地代理”

**Comment by Innition** [comment]
potplayer都可以连接webdev，手机端infuse连不上，气死了

**Comment by 10uder** [comment]
onecommander 用不了webdav, 映射网络驱动器那里也无法添加webdav链接

**Comment by ericvlog** [comment]
主机/路径 不要放 https / http 

```
test.org/dav
```

**Comment by oLerWT** [comment]
我也是使用v3.34.0的版本，使用的环境是群晖，网上找了一圈找到一个教程可以挂载，群晖File Station添加WebDAV的配置如下：
IP：127.0.0.1
端口：5244
路径：dav
账号：admin
密码：alist
代码页：Unicode(UTF-8)
配置文件名称：Alist
原文链接：https://www.cnblogs.com/saintyear/p/17214539.html

**Comment by noelesther** [comment]
115驱动不支持webdav吗？

**Comment by grm668** [comment]
failed get objs: failed to list objs: invalid character 'a' looking for beginning of value?   挂载115 播放不了
